fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialogBottomSheet.kt

```diff
@@ -0,0 +1,220 @@
+        add(
+            TranslationSwitchItem(
+                isEnabled = true,
+                stateChange = {},
+                hasDivider = true,
+                contentDescription = getSwitchContentDescription(
+                    stringResource(R.string.translation_option_bottom_sheet_always_translate),
+                    true,
+                ),
+                textLabel = stringResource(R.string.translation_option_bottom_sheet_always_translate),
+            ),
+        )
+        add(
+            TranslationSwitchItem(
+                isEnabled = false,
+                stateChange = {},
+                hasDivider = false,
+                contentDescription = getSwitchContentDescription(
+                    stringResource(
+                        id = R.string.translation_option_bottom_sheet_always_translate_in_language,
+                        formatArgs = arrayOf(stringResource(R.string.translation_option_bottom_sheet_spanish_language)),
+                    ),
+                    false,
+                ),
+                textLabel = stringResource(
+                    id = R.string.translation_option_bottom_sheet_always_translate_in_language,
+                    formatArgs = arrayOf(stringResource(R.string.translation_option_bottom_sheet_spanish_language)),
+                ),
+            ),
+        )
+        add(
+            TranslationSwitchItem(
+                isEnabled = true,
+                stateChange = {},
+                hasDivider = true,
+                contentDescription = getSwitchContentDescription(
+                    stringResource(
+                        id = R.string.translation_option_bottom_sheet_never_translate_in_language,
+                        formatArgs = arrayOf(stringResource(R.string.translation_option_bottom_sheet_spanish_language)),
+                    ),
+                    true,
+                ),
+                textLabel = stringResource(
+                    id = R.string.translation_option_bottom_sheet_never_translate_in_language,
+                    formatArgs = arrayOf(stringResource(R.string.translation_option_bottom_sheet_spanish_language)),
+                ),
+            ),
+        )
+        add(
+            TranslationSwitchItem(
+                isEnabled = true,
+                stateChange = {},
+                hasDivider = true,
+                contentDescription = getSwitchContentDescription(
+                    stringResource(R.string.translation_option_bottom_sheet_never_translate_site),
+                    true,
+                ),
+                textLabel = stringResource(R.string.translation_option_bottom_sheet_never_translate_site),
+            ),
+        )
+    }
+}
+
+/**
+ * Returns a content description for a switch list item based on the enabled value.
+ */
+@Composable
+fun getSwitchContentDescription(textSwitch: String, isEnabled: Boolean) =
+    if (isEnabled) {
+        "$textSwitch " + stringResource(
```

Comment:
We shouldn't be appending strings for localization. Please consider using parameterize strings.

Reply:
Done